### ISSUE TITLE
The parameter of the tracking loss

### ISSUE THREAD

**Issue by Haelles** [hustvl/QueryInst#36]
Thank you for your nice work!

Since the training code of QueryTrack is not released, I hope you can share the following training details with me:
1. What are the meaning of $\alpha_t$ and $\gamma$ in the formula(7) (the definition of the tracking loss)
2. Is there any document about your contrastive focal loss?

Thanks again.

**Comment by vealocia** [comment]
> Thank you for your nice work!
> 
> Since the training code of QueryTrack is not released, I hope you can share the following training details with me:
> 
> 1. What are the meaning of $\alpha_t$ and $\gamma$ in the formula(7) (the definition of the tracking loss)
> 2. Is there any document about your contrastive focal loss?
> 
> Thanks again.

$\alpha_t$ and $\gamma$ is the same as $\alpha$ and $\gamma$ in focal loss.

In form, our contrastive focal loss is a focal loss with softmax function (Eq.5-Eq.7).

**Comment by Haelles** [comment]
> > Thank you for your nice work!
> > Since the training code of QueryTrack is not released, I hope you can share the following training details with me:
> > 
> > 1. What are the meaning of $\alpha_t$ and $\gamma$ in the formula(7) (the definition of the tracking loss)
> > 2. Is there any document about your contrastive focal loss?
> > 
> > Thanks again.
> 
> $\alpha_t$ and $\gamma$ is the same as $\alpha$ and $\gamma$ in focal loss.
> 
> In form, our contrastive focal loss is a focal loss with softmax function (Eq.5-Eq.7).

Thank you!
By the way, I have another question. During the calculation of `losses['loss_cls']` in class DIIHead, the value of label_weight is always 1:

`label_weights[pos_inds] = pos_weight`
`label_weights[neg_inds] = 1.0`

However, when calculating `avg_factor`, only positive samples are included. Could you please tell me why? Thanks a lot!
